ccloud/beginner-cloud/start.sh

```diff
@@ -24,7 +24,7 @@ check_jq || exit 1
 # Create a new environment and specify it as the default
 ##################################################
 
-ENVIRONMENT_NAME="demo-script-env"
+ENVIRONMENT_NAME="demo-env-000000-beginner-cli"
```

Comment:
What does `000000` represent here?

An additional question, considering we are refactoring this bit, could it make sense to remove the `env` from `demo-env`?  Based on consideration that an environment already categorizes itself as an environment. So do we need the word "env" when naming environments, given that's what they are the name may be redundant.

Reply:
1. `xxxxxx` is the service account ID.  Since this tutorial is a bit different (env created before SA), it uses `000000`.
2. Agree, will remove `env`.  I don't think there's use in adding `ia` or `stack` though...unless there is concern about user-defined environments that start with `demo-`. WDYT?

Reply:
I'm good w/ just `demo` and no additional category. 👍

Reply:
@rspurgeon I decided to make it `ccloud-stack-` prefix.  Reasons:

1. Side-step opinion whether its purpose is demo/example/production/other
2. More explicit about how it was created